src/App.tsx

```
@@ -1,26 +1,42 @@
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
 import { Loader, TodoFilter, TodoList, TodoModal } from './components';
+import { useEffect, useState } from 'react';
+import { getTodos } from './api';
+import { useDispatch } from 'react-redux';
+import { todosSlice } from './features/todos';
+import { useAppSelector } from './app/hooks';
 
-export const App = () => (
-  <>
-    <div className="section">
-      <div className="container">
-        <div className="box">
-          <h1 className="title">Todos:</h1>
+export const App = () => {
+  const [loading, setLoading] = useState(true);
+  const dispatch = useDispatch();
+  const setTodos = todosSlice.actions.setTodos;
+  const currentTodo = useAppSelector(state => state.currentTodo);
 
-          <div className="block">
-            <TodoFilter />
-          </div>
+  useEffect(() => {
+    getTodos()
+      .then(todosFromServer => dispatch(setTodos(todosFromServer)))
+      .catch()
```

The `.catch()` block is empty, which means any errors from the `getTodos()` promise will be ignored. It's better to at least log the error or handle it appropriately to avoid silent failures and make debugging easier.